Parsers/ASimDns/Product parsers/vimDnsCiscoUmbrella.yaml

```diff
@@ -0,0 +1,113 @@
+Parser:
+  Title: Cisco Umbrella DNS Events
+  Version: '0.0'
+  LastUpdated: May 10, 2021
+Product:
+  Name: Cisco
+Normalization:
+  Schema: Dns
+  Version: '0.2.0'
+References:
+- Title: ASIM DNS Schema
+  Link: https://aka.ms/AzSentinelDnsDoc
+- Title: ASIM
+  Link: https://aka.ms/AzSentinelNormalization
+Description: |
+  This is a Query Parser that is used to map Cisco Umrella Dns Events (Cisco_Umbrella_dns_CL) to the Azure Sentinel Information Model Dns schema.
+ParserName: vimDnsCiscoUmbrella
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr
+    Type: string
+    Default: '*'
+  - Name: domain_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: responsecodename
+    Type: string
+    Default: '*'
+  - Name: dnsresponsename
```

Comment:
I think we may have missed on the name of this parameter, as it checks only for IPv4: maybe response_has_any_ipv4?